client/components/backup-retention-management/hooks.tsx

```diff
@@ -0,0 +1,73 @@
+import { useMemo } from '@wordpress/element';
+import { useSelector } from 'react-redux';
+import { useStorageText } from 'calypso/components/backup-storage-space/hooks';
+import getBackupCurrentSiteSize from 'calypso/state/rewind/selectors/get-backup-current-site-size';
+import getRewindBytesAvailable from 'calypso/state/rewind/selectors/get-rewind-bytes-available';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { RetentionRadioOptionType } from './consts';
```

Comment:
```suggestion
import type { RetentionRadioOptionType } from './consts';
```
Since we are importing type.